includes/Core/Util/Requires_Javascript_Trait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Require_Javascript_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Trait to display no javascript fallback message.
+ *
+ * @since 1.1.2
+ * @access private
+ * @ignore
+ */
+Trait Require_Javascript_Trait {
```

Comment:
`trait`should be in lowercase.

Reply:
Also, it looks like the trait name should only be `Require_Javascript`. cc @aaemnnosttv

Reply:
Most traits do have the `_Trait` suffix as part of the actual name but that may only be because they have a corresponding interface which would conflict otherwise.

The IB mentions it as a `Requires_Javascript` (not `Require_Javascript`) which is good I think.

Reply:
the `Migrate_Legacy_Keys` doesn't have the `_Trait` suffix that was mostly why I suggested removing it.

Reply:
Yep. It may be that as we add more traits we develop more of a well-defined convention going forward but I think it's okay without the suffix for now 👍